src/style.css

```diff
@@ -2,0 +1,51 @@
+@font-face {
+  src: url(../src/fonts/Avenir.ttc);
+  font-family: Avenir, sans-serif;
+}
+
+body {
+  font-family: Avenir, sans-serif;
+}
+
+input {
+  width: 100%;
+  font-weight: 300;
+  line-height: 21.86px;
+}
+
+.search-field {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  background-image: url(../src/images/Search.svg);
```

Comment:
Ensure that the image file path is correct. The path `../src/images/Search.svg` should be verified to ensure the background image loads correctly.